web/src/lib/data-hooks/useUserData.ts

```diff
@@ -24,6 +26,11 @@ export const useUserData = (): UseUserDataResponse => {
     },
   });
   const dataExists = !!data;
+  const { setOperatingPhaseId } = useContext(IntercomContext);
+
+  useEffect(() => {
+    setOperatingPhaseId(data?.profileData.operatingPhase);
+  }, [data?.profileData.operatingPhase]);
```

Comment:
This works, but another option could be to make use the of `onProfileDataChange` function down below, which already runs every time something changes in profileData. It would run more frequently, but would also group related behavior (things that update contexts when something in profile data is changed)

Reply:
There seems to be a lag in the operatingPhase when I use `onProfileDataChange`.